include/fw_info.rst

```diff
@@ -29,6 +29,9 @@ It includes the following information:
 * The single, monotonically increasing version number of the image.
 * The address through which to boot into the firmware (the vector table address).
   This address is not necessarily the start of the image.
+* A value that must be 0xFFFFFFFF.
+  If this is set to 0 (e.g. by the bootloader), the image can quickly be established as invalid.
+  The bootloader will set this to 0 when the image fails validation in some way, so a costly validation doesn't need to be performed on every boot.
```

Comment:
```suggestion
  The bootloader will set this to 0 when the image fails validation in some way, so that there is no need to perform a costly validation on every boot.
```